app/models/manageiq/providers/amazon/inventory/parser/cloud_manager.rb

```diff
@@ -247,7 +247,7 @@ def parse_image(image, is_public)
       :template              => true,
       # the is_public flag here avoids having to make an additional API call
       # per image, since we already know whether it's a public image
-      :publicly_available    => is_public,
+      :publicly_available    => image['public'],
```

Comment:
I wanted to answer, of course this is tested in the refresh and it passes. But this attribute is not tested in the refresh :-D Let me add it.